salsa_dancing_molecules/startup/startup_script.py

```diff
@@ -0,0 +1,28 @@
+"""Program for handling startup tasks."""
+from .prepare_workspace import do_preparations
+from .use_custom_materials import materials_to_pickles
+from .config_module import read_configuration
+from .generate_json import convert_to_json
+import os
+
+
+def start(conf_path):
+    """Run the different modules of the startup program.
```

Comment:
Please add a brief description of what the result of running these modules will be. It doesn't have to be detailed, but at least give an idea of what will happen.